lib/backend/manager.go

```diff
@@ -139,3 +144,18 @@ func (m *Manager) GetClient(namespace string) (Client, error) {
 	}
 	return nil, ErrNamespaceNotFound
 }
+
+// IsReady returns whether the backends are ready (reachable).
+// A backend must be explicitly configured as required for readiness to be checked.
+func (m *Manager) IsReady() (bool, error) {
+	for _, b := range m.backends {
+		if !b.mustReady {
+			continue
+		}
+		_, err := b.client.Stat(isReadyNamespace, isReadyName)
+		if err != nil && err != backenderrors.ErrBlobNotFound {
```

Comment:
So is backenderrors.ErrBlobNotFound return for all backends when the blob doesn't exist?

Reply:
Yes. It's required for the interface. Either way, I've changed the code not to use it anymore.